CMakeLists.txt

```diff
@@ -149,6 +149,7 @@ endif()
 # I18N
 
 if (I18N AND GETTEXT_FOUND AND LIBINTL_LIB_FOUND)
+  include_directories(${LIBINTL_INCLUDE_DIR})
```

Comment:
Please make that a system include with `include_directories(SYSTEM ...)`. It's not likely that the libintl headers are going to introduce warnings on their own, but it's a bit more correct.